package.json

```diff
@@ -65,6 +65,9 @@
       "plugin:import/typescript"
     ],
     "rules": {
+      "@typescript-eslint/prefer-optional-chain": "off",
```

Comment:
Stylistic rule whose rational is not suitable for a lint rule (false-positive rate needs to be next to zero).